frontend/src/scenes/toolbar-launch/AuthorizedUrls.tsx

```diff
@@ -122,12 +122,13 @@ export function AuthorizedUrls({ pageKey, actionId }: AuthorizedUrlsTableInterfa
                                                 Suggestion
                                             </LemonTag>
                                         )}
-                                        <Typography.Text
-                                            ellipsis={{ tooltip: keyedAppURL.url }}
-                                            className="text-muted-alt flex-1"
-                                        >
-                                            {keyedAppURL.url}
-                                        </Typography.Text>
+                                        <div className="flex-1">
+                                            <LemonButton tooltip="Launch toolbar" href={launchUrl(keyedAppURL.url)}>
+                                                <Typography.Text ellipsis={{ tooltip: keyedAppURL.url }}>
+                                                    {keyedAppURL.url}
+                                                </Typography.Text>
+                                            </LemonButton>
+                                        </div>
```

Comment:
This change wasn't documented in the PR description. It contains several design changes, which we try to communicate with screencasts to make it easy to follow along.

In any case I'm not sure it's an improvement:

![2022-08-23 11 21 29](https://user-images.githubusercontent.com/53387/186122100-04f28977-baf0-439c-a0d3-959d02abea2d.gif)

- The tooltip is positioned so far from the text's it's tippin'. I don't know where I'm supposed to look when the it comes up
- I'm not sure if simple bold black text should be used as a button's label.
- I don't think a tooltip is the best way to communicate this either: you need to realise it's a button and hover it to see what it does. It should be obvious where to click without moving your mouse. This also breaks down if you use a tablet.

The rest looks good, but gonna "request changes" for this. Easiest to just revert and deal with this in a new PR.

My alternative suggestion would be something like this:

<img width="1473" alt="image" src="https://user-images.githubusercontent.com/53387/186124399-39fc47b9-52e4-4f1f-b1bf-e75307c024ec.png">

Reply:
@mariusandra done, can you stamp please as I can't merge this now